R/MixedModelsMessages.R

```diff
@@ -15,24 +15,21 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-.mmMessageInterpretability <-
-  gettext("The intercept corresponds to the (unweighted) grand mean; for each factor with k levels, k - 1 parameters are estimated with sum contrast coding. Consequently, the estimates cannot be directly mapped to factor levels. Use estimated marginal means for obtaining estimates for each factor level/design cell or their differences.")
-.mmMessageInterpretabilityBayesian <-
-  gettext("The intercept corresponds to the (unweighted) grand mean; for each factor with k levels, k - 1 parameters are estimated with orthonormal coding proposed by Rouder et al. (2012). Consequently, the estimates cannot be directly mapped to factor levels. Use estimated marginal means for obtaining estimates for each factor level/design cell or their differences.")
-.mmMessageSingularFit <-
-  gettext("Model fit is singular. Specified random effects parameters (random intercepts and random slopes) cannot be estimated from the available data. Carefully reduce the random effects structure, but this practice might inflate the reported p-value, and invalidates the analysis.")
-.mmMessageVovkSellke <-
-  gettextf("Vovk-Sellke Maximum <em>p</em>-Ratio: Based on a two-sided <em>p</em>-value, the maximum possible odds in favor of H%1$s over H%2$s equals 1/(-e <em>p</em> log(<em>p</em>)) for <em>p</em> %3$s .37 (Sellke, Bayarri, & Berger, 2001).","\u2081","\u2080","\u2264")
-.mmMessageNumericalProblems <-
-  gettext("Numerical problems with the maximum-likelihood estimate (e.g., gradients too large). This may indicate that the specified random effects parameters (random intercepts and random slopes) cannot be estimated from the available data. Consider carefully reducing the random effects structure, but be aware this may induce unknown risks of anti-conservative results (i.e., p-values might be lower than nominal).")
-.mmMessageDFdisabled <-
-  gettext("Estimation of degrees of freedom disabled (i.e., asymptotic results shown), because the number of observations is large. To force estimation, check corresponding option.")
-.mmMessageResponse    <- gettext("Results are on the response scale.")
-.mmMessageNotResponse <-
-  gettext("Results are not on the response scale and might be misleading.")
-.mmMessageANOVAtype     <- function(type) {
-  gettextf("Type %s Sum of Squares",type)
-}
+#No need for gettext on next line as they are wider than even my big ass screen anyway
```

Comment:
time to get a bigger screen then. Could we get [this](https://www.youtube.com/watch?v=tURZaUyUoOM&ab_channel=LinusTechTips) as a "work equipment"?

Reply:
Hahaha, I don't think I can fit that in my workspace here :p